samples/sample/src/main/java/com/microsoft/jfr/App.java

```diff
@@ -0,0 +1,67 @@
+package com.microsoft.jfr;
+
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanServerConnection;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.math.BigInteger;
+import java.nio.file.Paths;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.RecursiveAction;
+
+public class App 
+{
+    private static final ForkJoinPool threadPool = new ForkJoinPool();
+    private static final int range = 100000;
+
+    static class Fibonacci extends RecursiveAction {
+
+        public BigInteger NthFibonacci(int n) {
+            BigInteger[] fibonacciNumbers = new BigInteger[n+2];
+            fibonacciNumbers[0] = BigInteger.ZERO;
+            fibonacciNumbers[1] = BigInteger.ONE;
+            for(int i = 2; i <= n; i++){
+                fibonacciNumbers[i] = fibonacciNumbers[i-1].add(fibonacciNumbers[i-2]);
+            }
+            return fibonacciNumbers[n];
+        }
+
+        @Override
+        protected void compute() {
+            // Do some computation that heats up the CPU
+            for (int i = 1; i<=range; i++){
+                System.out.println( i + "th Fibonacci number: " + NthFibonacci(i).toString());
+            }
+        }
+    }
+
+    public static void main( String[] args )
+    {
+        // Create a connection to the mBean server
```

Comment:
A reader can get the 'what' from reading the code. The comments should be more about the "why".

Reply:
I have changed the comment based on what I read and understood in the last few days, and also based on what I read from the library code. Let me know if we can make it better